src/pages/networks/forms/NetworkForm.tsx

```diff
@@ -0,0 +1,713 @@
+                <Input
+                  {...getFormProps("ipv4_address")}
+                  type="text"
+                  label="IPv4 address"
+                  help="IPv4 address for the bridge (use none to turn off IPv4 or auto to generate a new random unused subnet) (CIDR)"
+                />
+              )}
+              {formik.values.ipv4_address !== "none" && (
+                <Select
+                  {...getFormProps("ipv4_nat")}
+                  label="IPv4 NAT"
+                  help="Network address translation for IPv4"
+                  options={optionEnableDisable}
+                />
+              )}
+              {formik.values.bridge_mode !== "fan" && (
+                <Input
+                  {...getFormProps("ipv6_address")}
+                  type="text"
+                  label="IPv6 address"
+                  help="IPv6 address for the bridge (use none to turn off IPv6 or auto to generate a new random unused subnet) (CIDR)"
+                />
+              )}
+              {formik.values.ipv6_address !== "none" && (
+                <Select
+                  {...getFormProps("ipv6_nat")}
+                  label="IPv6 NAT"
+                  help="Network address translation for IPv6"
+                  options={optionEnableDisable}
+                />
+              )}
+            </React.Fragment>
+          )}
+          {[BRIDGE, ADVANCED_OVN].includes(section) && (
+            <>
+              <Input
+                {...getFormProps("bridge_mtu")}
+                type="text"
+                label="MTU"
+                help="Bridge MTU (default varies if tunnel or fan setup)"
+              />
+              <Input
+                {...getFormProps("bridge_hwaddr")}
+                type="text"
+                label="MAC address"
+                help="MAC address for the bridge"
+              />
+              {formik.values.type.startsWith("bridge") && (
+                <>
+                  <Select
+                    {...getFormProps("bridge_driver")}
+                    label="Bridge Driver"
+                    help="Bridge driver: native or openvswitch"
+                    options={[
+                      {
+                        label: "Select option",
+                        value: "",
+                        disabled: true,
+                      },
+                      {
+                        label: "Native",
+                        value: "native",
+                      },
+                      {
+                        label: "Openvswitch",
+                        value: "openvswitch",
+                      },
+                    ]}
+                  />
+                  <Input
+                    {...getFormProps("bridge_external_interfaces")}
+                    type="text"
+                    label="Bridge external interfaces"
+                    help="Comma-separated list of unconfigured network interfaces to include in the bridge"
```

Comment:
I think all the "comma-separated list"s should be textareas. This is what we also do in the "Networks" section of a project configuration. We should keep a consistent design with that.